assets/js/components/Work/CardItem.jsx

```diff
@@ -1,7 +1,12 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
-import { setVisibilityClass, formatDate } from "../../services/helpers";
+import {
+  setVisibilityClass,
+  formatDate,
+  getImageUrl,
+} from "../../services/helpers";
```

Comment:
Again, not a huge deal, but we can start using this pattern for aliasing relative paths when importing nested components:

```
import {
  setVisibilityClass,
  formatDate,
  getImageUrl,
} from "@js/services/helpers";
```